2020-02-08 11:14:14,794 ERROR 84589 [-/127.0.0.1/-/1ms GET /] nodejs.PROTOCOL_SEQUENCE_TIMEOUTError: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at emitNone (events.js:106:13)
    at Handshake.emit (events.js:208:7)
    at Handshake._onTimeout (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
    --------------------
    at Protocol._enqueue (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
code: "PROTOCOL_SEQUENCE_TIMEOUT"
fatal: true
timeout: 10000
name: "PROTOCOL_SEQUENCE_TIMEOUTError"
pid: 84589
hostname: lixindeMacBook-Pro.local

2020-02-08 15:35:28,248 ERROR 85985 [-/127.0.0.1/-/0ms GET /] nodejs.PROTOCOL_SEQUENCE_TIMEOUTError: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at emitNone (events.js:106:13)
    at Handshake.emit (events.js:208:7)
    at Handshake._onTimeout (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
    --------------------
    at Protocol._enqueue (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
code: "PROTOCOL_SEQUENCE_TIMEOUT"
fatal: true
timeout: 10000
name: "PROTOCOL_SEQUENCE_TIMEOUTError"
pid: 85985
hostname: lixindeMacBook-Pro.local

2020-02-09 10:40:12,597 ERROR 24958 [-/127.0.0.1/-/0ms GET /] nodejs.ER_NOT_SUPPORTED_AUTH_MODEError: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Handshake.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
code: "ER_NOT_SUPPORTED_AUTH_MODE"
errno: 1251
sqlMessage: "Client does not support authentication protocol requested by server; consider upgrading MySQL client"
sqlState: "08004"
fatal: true
name: "ER_NOT_SUPPORTED_AUTH_MODEError"
pid: 24958
hostname: lixindeMacBook-Pro.local

2020-02-09 10:45:47,870 ERROR 25096 [-/127.0.0.1/-/0ms GET /] nodejs.ER_NOT_SUPPORTED_AUTH_MODEError: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
    at Handshake.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Protocol._enqueue (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
code: "ER_NOT_SUPPORTED_AUTH_MODE"
errno: 1251
sqlMessage: "Client does not support authentication protocol requested by server; consider upgrading MySQL client"
sqlState: "08004"
fatal: true
name: "ER_NOT_SUPPORTED_AUTH_MODEError"
pid: 25096
hostname: lixindeMacBook-Pro.local

2020-02-09 13:42:32,330 ERROR 25750 [-/::1/-/13ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:8:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 25750
hostname: lixindeMacBook-Pro.local

2020-02-09 13:43:00,350 ERROR 25921 [-/::1/-/13ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:8:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 25921
hostname: lixindeMacBook-Pro.local

2020-02-09 13:57:20,914 ERROR 26019 [-/::1/-/22ms GET /todo/list] nodejs.TypeError: this.app.mysql.query is not a function
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:43)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 26019
hostname: lixindeMacBook-Pro.local

2020-02-09 13:57:41,161 ERROR 26036 [-/::1/-/21ms GET /todo/list] nodejs.TypeError: this.app.mysql.query is not a function
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:43)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 26036
hostname: lixindeMacBook-Pro.local

2020-02-09 14:00:27,773 ERROR 26070 [-/::1/-/18ms GET /todo/list] nodejs.ReferenceError: app is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:21)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 26070
hostname: lixindeMacBook-Pro.local

2020-02-09 14:00:49,511 ERROR 26071 [-/::1/-/25ms GET /todo/list] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'db1' at line 1
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: db1
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'db1' at line 1"
sqlState: "42000"
index: 0
sql: "db1"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 26071
hostname: lixindeMacBook-Pro.local

2020-02-09 14:28:25,928 ERROR 26185 [-/::1/-/60ms GET /todo/list] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: test
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1"
sqlState: "42000"
index: 0
sql: "test"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 26185
hostname: lixindeMacBook-Pro.local

2020-02-09 14:28:48,493 ERROR 26186 [-/::1/-/18ms GET /todo/list] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: test
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1"
sqlState: "42000"
index: 0
sql: "test"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 26186
hostname: lixindeMacBook-Pro.local

2020-02-09 14:29:00,137 ERROR 26186 [-/::1/-/4ms GET /todo/list] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: test
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1"
sqlState: "42000"
index: 0
sql: "test"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 26186
hostname: lixindeMacBook-Pro.local

2020-02-09 14:33:21,460 ERROR 26222 [-/::1/-/17ms GET /todo/list] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: test
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'test' at line 1"
sqlState: "42000"
index: 0
sql: "test"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 26222
hostname: lixindeMacBook-Pro.local

2020-02-09 15:34:09,419 ERROR 26432 [-/::1/-/15ms GET /todo/list] nodejs.ER_EMPTY_QUERYError: ER_EMPTY_QUERY: Query was empty
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: undefined
code: "ER_EMPTY_QUERY"
errno: 1065
sqlMessage: "Query was empty"
sqlState: "42000"
index: 0
sql: undefined
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_EMPTY_QUERYError"
pid: 26432
hostname: lixindeMacBook-Pro.local

2020-02-09 15:34:40,746 ERROR 26433 [-/::1/-/29ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:9:14)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 26433
hostname: lixindeMacBook-Pro.local

2020-02-09 15:48:24,774 ERROR 26541 [-/::1/-/16ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:9:14)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 26541
hostname: lixindeMacBook-Pro.local

2020-02-09 15:49:16,861 ERROR 26553 [-/::1/-/27ms GET /todo/list] nodejs.ER_EMPTY_QUERYError: ER_EMPTY_QUERY: Query was empty
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: undefined
code: "ER_EMPTY_QUERY"
errno: 1065
sqlMessage: "Query was empty"
sqlState: "42000"
index: 0
sql: undefined
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_EMPTY_QUERYError"
pid: 26553
hostname: lixindeMacBook-Pro.local

2020-02-09 15:54:13,989 ERROR 26587 [-/::1/-/12ms GET /todo/list] nodejs.ReferenceError: sql is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:6:42)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 26587
hostname: lixindeMacBook-Pro.local

2020-02-09 19:37:15,438 ERROR 27303 [-/::1/-/10ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:9:14)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 27303
hostname: lixindeMacBook-Pro.local

2020-02-09 23:39:18,341 ERROR 27303 [-/::1/-/5ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:9:14)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 27303
hostname: lixindeMacBook-Pro.local

2020-02-09 23:47:50,821 ERROR 28171 [-/::1/-/11ms GET /todo/list] nodejs.TypeError: this.app.mysql.insert is not a function
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:41)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28171
hostname: lixindeMacBook-Pro.local

2020-02-09 23:48:02,022 ERROR 28171 [-/::1/-/1ms GET /todo/list] nodejs.TypeError: this.app.mysql.insert is not a function
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:41)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28171
hostname: lixindeMacBook-Pro.local

2020-02-09 23:48:19,746 ERROR 28173 [-/::1/-/11ms GET /todo/list] nodejs.TypeError: this.app.mysql.insert is not a function
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:41)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28173
hostname: lixindeMacBook-Pro.local

2020-02-09 23:48:33,962 ERROR 28173 [-/::1/-/1ms GET /todo/list] nodejs.TypeError: this.app.mysql.insert is not a function
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:5:41)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28173
hostname: lixindeMacBook-Pro.local

2020-02-09 23:48:48,327 ERROR 28180 [-/::1/-/15ms GET /todo/list] nodejs.ReferenceError: result is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:9:27)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28180
hostname: lixindeMacBook-Pro.local

2020-02-09 23:50:49,042 ERROR 28195 [-/::1/-/13ms GET /todo/list] nodejs.ReferenceError: result is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:9:27)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28195
hostname: lixindeMacBook-Pro.local

2020-02-10 00:02:08,806 ERROR 28299 [-/::1/-/30ms GET /todo/list] nodejs.ReferenceError: result is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:12:27)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28299
hostname: lixindeMacBook-Pro.local

2020-02-10 00:02:21,240 ERROR 28300 [-/::1/-/16ms GET /todo/list] nodejs.ReferenceError: result is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:12:27)
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28300
hostname: lixindeMacBook-Pro.local

2020-02-10 00:05:03,915 ERROR 28324 [-/::1/-/99ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:14:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28324
hostname: lixindeMacBook-Pro.local

2020-02-10 00:05:45,572 ERROR 28332 [-/::1/-/23ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:14:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28332
hostname: lixindeMacBook-Pro.local

2020-02-10 00:10:17,363 ERROR 28363 [-/::1/-/10ms GET /todo/list] nodejs.SyntaxError: Identifier 'result' has already been declared
    at TodoController.todoList (/Users/lixin/project/egg-example/app/controller/todo.js:5:43)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at /Users/lixin/project/egg-example/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28363
hostname: lixindeMacBook-Pro.local

2020-02-10 00:10:36,027 ERROR 28366 [-/::1/-/16ms GET /todo/list] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'id' in 'field list'
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: INSERT INTO `todo_list`(`id`, `name`, `otherField`) VALUES(123, 'fengmk2', 'other field value')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'id' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `todo_list`(`id`, `name`, `otherField`) VALUES(123, 'fengmk2', 'other field value')"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 28366
hostname: lixindeMacBook-Pro.local

2020-02-10 00:10:44,607 ERROR 28366 [-/::1/-/2ms GET /todo/list] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'id' in 'field list'
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: INSERT INTO `todo_list`(`id`, `name`, `otherField`) VALUES(123, 'fengmk2', 'other field value')
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'id' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `todo_list`(`id`, `name`, `otherField`) VALUES(123, 'fengmk2', 'other field value')"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 28366
hostname: lixindeMacBook-Pro.local

2020-02-10 00:14:08,880 ERROR 28397 [-/::1/-/18ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:21:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28397
hostname: lixindeMacBook-Pro.local

2020-02-10 00:14:30,958 ERROR 28399 [-/::1/-/13ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:21:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28399
hostname: lixindeMacBook-Pro.local

2020-02-10 00:16:35,549 ERROR 28417 [-/::1/-/16ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:21:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28417
hostname: lixindeMacBook-Pro.local

2020-02-10 00:16:55,769 ERROR 28424 [-/::1/-/14ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:21:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28424
hostname: lixindeMacBook-Pro.local

2020-02-10 00:18:52,490 ERROR 28449 [-/::1/-/19ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:23:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28449
hostname: lixindeMacBook-Pro.local

2020-02-10 00:19:43,391 ERROR 28454 [-/::1/-/20ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28454
hostname: lixindeMacBook-Pro.local

2020-02-10 00:20:03,953 ERROR 28461 [-/::1/-/36ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28461
hostname: lixindeMacBook-Pro.local

2020-02-10 00:20:12,525 ERROR 28461 [-/::1/-/3ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28461
hostname: lixindeMacBook-Pro.local

2020-02-10 00:21:03,975 ERROR 28468 [-/::1/-/32ms GET /todo/list] nodejs.ReferenceError: getSuccess is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:23:5)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28468
hostname: lixindeMacBook-Pro.local

2020-02-10 00:21:27,581 ERROR 28470 [-/::1/-/20ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28470
hostname: lixindeMacBook-Pro.local

2020-02-10 00:21:49,397 ERROR 28477 [-/::1/-/27ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28477
hostname: lixindeMacBook-Pro.local

2020-02-10 00:22:13,670 ERROR 28480 [-/::1/-/31ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28480
hostname: lixindeMacBook-Pro.local

2020-02-10 00:26:46,339 ERROR 28528 [-/::1/-/21ms GET /todo/list] nodejs.ReferenceError: result_1 is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:22:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28528
hostname: lixindeMacBook-Pro.local

2020-02-10 00:27:03,119 ERROR 28536 [-/::1/-/16ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:27:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28536
hostname: lixindeMacBook-Pro.local

2020-02-10 00:32:45,179 ERROR 28575 [-/::1/-/20ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:30:14)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 28575
hostname: lixindeMacBook-Pro.local

2020-02-10 19:52:20,141 ERROR 33960 [-/::1/-/15ms GET /todo/list] nodejs.ReferenceError: todo is not defined
    at TodoService.list (/Users/lixin/project/egg-example/app/service/todo.js:30:14)
    at process._tickCallback (internal/process/next_tick.js:68:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 33960
hostname: lixindeMacBook-Pro.local

2020-02-10 21:04:45,226 WARN 34578 [-/::1/-/28ms POST /todo/list] missing csrf token. See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范
2020-02-10 21:04:45,228 WARN 34578 [-/::1/-/29ms POST /todo/list] nodejs.ForbiddenError: missing csrf token
    at Object.throw (/Users/lixin/project/egg-example/node_modules/koa/lib/context.js:97:11)
    at Object.assertCsrf (/Users/lixin/project/egg-example/node_modules/egg-security/app/extend/context.js:153:19)
    at csrf (/Users/lixin/project/egg-example/node_modules/egg-security/lib/middlewares/csrf.js:33:9)
    at dispatch (/Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:42:32)
    at /Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at session (/Users/lixin/project/egg-example/node_modules/koa-session/index.js:41:13)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at overrideMethod (/Users/lixin/project/egg-example/node_modules/koa-override/index.js:38:12)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
message: "missing csrf token"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 34578
hostname: lixindeMacBook-Pro.local

2020-02-10 21:05:27,498 WARN 34578 [-/::1/-/1ms POST /todo/list] missing csrf token. See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范
2020-02-10 21:05:27,498 WARN 34578 [-/::1/-/1ms POST /todo/list] nodejs.ForbiddenError: missing csrf token
    at Object.throw (/Users/lixin/project/egg-example/node_modules/koa/lib/context.js:97:11)
    at Object.assertCsrf (/Users/lixin/project/egg-example/node_modules/egg-security/app/extend/context.js:153:19)
    at csrf (/Users/lixin/project/egg-example/node_modules/egg-security/lib/middlewares/csrf.js:33:9)
    at dispatch (/Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:42:32)
    at /Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at session (/Users/lixin/project/egg-example/node_modules/koa-session/index.js:41:13)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at overrideMethod (/Users/lixin/project/egg-example/node_modules/koa-override/index.js:38:12)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
message: "missing csrf token"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 34578
hostname: lixindeMacBook-Pro.local

2020-02-10 23:14:15,307 WARN 35107 [-/::1/-/26ms POST /todo/list] missing csrf token. See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范
2020-02-10 23:14:15,335 WARN 35107 [-/::1/-/50ms POST /todo/list] nodejs.ForbiddenError: missing csrf token
    at Object.throw (/Users/lixin/project/egg-example/node_modules/koa/lib/context.js:97:11)
    at Object.assertCsrf (/Users/lixin/project/egg-example/node_modules/egg-security/app/extend/context.js:153:19)
    at csrf (/Users/lixin/project/egg-example/node_modules/egg-security/lib/middlewares/csrf.js:33:9)
    at dispatch (/Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:42:32)
    at /Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at session (/Users/lixin/project/egg-example/node_modules/koa-session/index.js:41:13)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at overrideMethod (/Users/lixin/project/egg-example/node_modules/koa-override/index.js:38:12)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
message: "missing csrf token"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 35107
hostname: lixindeMacBook-Pro.local

2020-02-10 23:15:08,139 WARN 35125 [-/::1/-/14ms POST /todo/list] missing csrf token. See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范
2020-02-10 23:15:08,151 WARN 35125 [-/::1/-/18ms POST /todo/list] nodejs.ForbiddenError: missing csrf token
    at Object.throw (/Users/lixin/project/egg-example/node_modules/koa/lib/context.js:97:11)
    at Object.assertCsrf (/Users/lixin/project/egg-example/node_modules/egg-security/app/extend/context.js:153:19)
    at csrf (/Users/lixin/project/egg-example/node_modules/egg-security/lib/middlewares/csrf.js:33:9)
    at dispatch (/Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:42:32)
    at /Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at session (/Users/lixin/project/egg-example/node_modules/koa-session/index.js:41:13)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at overrideMethod (/Users/lixin/project/egg-example/node_modules/koa-override/index.js:38:12)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
message: "missing csrf token"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 35125
hostname: lixindeMacBook-Pro.local

2020-02-10 23:20:08,297 WARN 35125 [-/::1/-/0ms POST /todo/list] missing csrf token. See https://eggjs.org/zh-cn/core/security.html#安全威胁csrf的防范
2020-02-10 23:20:08,298 WARN 35125 [-/::1/-/1ms POST /todo/list] nodejs.ForbiddenError: missing csrf token
    at Object.throw (/Users/lixin/project/egg-example/node_modules/koa/lib/context.js:97:11)
    at Object.assertCsrf (/Users/lixin/project/egg-example/node_modules/egg-security/app/extend/context.js:153:19)
    at csrf (/Users/lixin/project/egg-example/node_modules/egg-security/lib/middlewares/csrf.js:33:9)
    at dispatch (/Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:42:32)
    at /Users/lixin/project/egg-example/node_modules/egg-security/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at session (/Users/lixin/project/egg-example/node_modules/koa-session/index.js:41:13)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
    at overrideMethod (/Users/lixin/project/egg-example/node_modules/koa-override/index.js:38:12)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa/node_modules/koa-compose/index.js:42:32)
message: "missing csrf token"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 35125
hostname: lixindeMacBook-Pro.local

2020-02-10 23:26:30,891 ERROR 35417 [-/::1/-/23ms POST /todo/list] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'status' in 'field list'
    at Query.Sequence._packetToError (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/lixin/project/egg-example/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/lixin/project/egg-example/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    --------------------
    at Pool.query (/Users/lixin/project/egg-example/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/lixin/project/egg-example/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/client.js:34:20)
    at RDSClient.proto.query (/Users/lixin/project/egg-example/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/lixin/project/egg-example/node_modules/co/index.js:65:19)
    at /Users/lixin/project/egg-example/node_modules/co/index.js:54:5
    sql: INSERT INTO `todo_list`(`name`, `status`) VALUES(NULL, 1)
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'status' in 'field list'"
sqlState: "42S22"
index: 0
sql: "INSERT INTO `todo_list`(`name`, `status`) VALUES(NULL, 1)"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35417
hostname: lixindeMacBook-Pro.local

2020-02-10 23:28:00,602 ERROR 35437 [-/::1/-/21ms POST /todo/list] nodejs.ReferenceError: result is not defined
    at TodoService.creatTodoItem (/Users/lixin/project/egg-example/app/service/todo.js:16:27)
    at process._tickCallback (internal/process/next_tick.js:68:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 35437
hostname: lixindeMacBook-Pro.local

2020-02-10 23:35:24,883 ERROR 35500 [-/::1/-/112ms POST /todo/list] nodejs.TypeError: Cannot read property 'name' of undefined
    at TodoController.creatTodoList (/Users/lixin/project/egg-example/app/controller/todo.js:11:31)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at /Users/lixin/project/egg-example/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 35500
hostname: lixindeMacBook-Pro.local

2020-02-10 23:40:21,831 ERROR 35550 [-/::1/-/29ms POST /todo/list] nodejs.TypeError: Cannot read property 'name' of undefined
    at TodoController.creatTodoList (/Users/lixin/project/egg-example/app/controller/todo.js:11:34)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/lixin/project/egg-example/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at next (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:45:18)
    at /Users/lixin/project/egg-example/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/lixin/project/egg-example/node_modules/koa-compose/index.js:44:32)
    at /Users/lixin/project/egg-example/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 35550
hostname: lixindeMacBook-Pro.local

